### DISCUSSION TITLE
刚了解到推出了一个百川模型（baichuan-7B），模型性能好于chatglm6B。

### DISCUSSION THREAD

**Discussion by shmily91** [wenda-LLM/wenda#387] [Ideas]
跟作者分享一下，看看需不需要加入这个模型

**Comment by l15y** [comment]
项目已支持，但性能离chatglm6B有很大差距。虚假宣传

**Reply by shmily91** [reply]
哈哈原来这样子，作者速度好快喔～赞👍

**Reply by jnmaomao** [reply]
原始模型是没有经过对话微调的，可以试试这个：
https://huggingface.co/hiyouga/baichuan-7b-sft
